flakeModules/agenix-shell.nix

```diff
@@ -229,11 +230,6 @@ in {
 
           printf 1>&2 -- '[agenix] decrypting secret %q from %q to %q...\n' ${lib.escapeShellArgs [name secret.file]} "$__agenix_shell_secret_path"
 
-          # shellcheck disable=SC2193
-          if [ "$__agenix_shell_secret_path" != "$__agenix_shell_secrets_path/${secret.name}" ]; then
-            mkdir -p "$(dirname "$__agenix_shell_secret_path")"
-          fi
-
```

Comment:
Why do you say this is unused?

Reply:
From my understanding:

Reply:
Yeah you are right, I also can't understand what could be the reason for that. Probably it was useful in a previous version and we forgot to remove it.